stdlib/toolchain/Compatibility56/Overrides.cpp

```diff
@@ -42,6 +42,7 @@ struct ConcurrencyOverrideSection {
 ConcurrencyOverrideSection Swift56ConcurrencyOverrides
 __attribute__((used, section("__DATA,__s_async_hook"))) = {
   .version = 0,
+  .task_create_common = swift56override_swift_task_create_common,
```

Comment:
Since this override doesn't do anything on 32-bit, do we want to move/copy the `#if` here and not override at all in that case?

Reply:
The `#if`should  be around `.task_create_common = swift56override_swift_task_create_common,`, not `future_wait`.